### DISCUSSION TITLE
Monitoring page grid filterting will not scroll back to wanted grid

### DISCUSSION THREAD

**Discussion by Dreimus** [PrestaShop/PrestaShop#34928] [Ideas]
### Prerequisites

- [X] I understand and accept the project's [code of conduct](https://github.com/PrestaShop/PrestaShop/blob/develop/CODE_OF_CONDUCT.md).
- [X] I have already [searched in existing issues](https://github.com/PrestaShop/PrestaShop/issues?q=is%3Aissue+label%3ABug) and found no previous report of this bug.

### Describe the bug and add attachments

Monitoring view filtering will always scroll you at the page top as you filter results. This is pretty frustrating as you have to scroll for each page view.

### Expected behavior

What is expected : 
- When you apply a filter on a grid, you expect, at the page reload, to be scrolled back to the filtered grid ( in our case, the second grid : "List of products with combinations but without available quantities for sale" )

### Steps to reproduce

How to : 
- Have a shop with multiple empty categories, and products with combinations but without available quantities for sale ( at least 20 )
- Limit items per page to 10
- Go to the second grid : "List of products with combinations but without available quantities for sale" and go to page 2 of the grid
- The page reloads, you are now on top of the page.

### PrestaShop version(s) where the bug happened

1.7.7.4+

### Your company or customer's name goes here (if applicable).

Evolutive

**Comment by AureRita** [comment]
Hi @Dreimus 

Thank you for your report, I think this one is more than a required feature that you want than a bug find ? 

If your are agree with me, we can change this into a discussion with the Pm to know if we'll add it to our backlog 

If it's a misundersting from my part, can you give us a video to show you issue ? 

Waiting for your feedback

**Comment by Dreimus** [comment]
Hello @AureRita , 

Yeah it's probably more a feature that a bug, could be a UX enhancement.
